tests/.ignore_lifespan

```diff
@@ -1,3 +0,0 @@
-[coverage:run]
-omit =
-  starlette/middleware/lifespan.py
```

Comment:
I found it very confusing to have this instead of `.coveragerc` - moved it to setup.cfg.